packages/cli/src/commands/preview-docs/preview-server/preview-server.ts

```diff
@@ -73,7 +73,7 @@ export default async function startPreviewServer(
           'Content-Type': 'text/html',
         },
       );
-    } else if (request.url === '/openapi.json') {
+    } else if (request.url?.endsWith('/openapi.json')) {
```

Comment:
Is it always requested from the root?

Reply:
I think we should change `openapi.json` to `/openapi.json` here: https://github.com/Redocly/openapi-cli/blob/master/packages/cli/src/commands/preview-docs/preview-server/preview-server.ts#L49

Reply:
yes, we can fix it there.